tasks/main.yml

```diff
@@ -137,6 +137,7 @@
         auth: "{{ os_images_auth }}"
         cacert: "{{ os_images_cacert | default(omit) }}"
         interface: "{{ os_images_interface | default(omit, true) }}"
+        region: "{{ os_images_region | default(omit) }}"
```

Comment:
Docs say it's region_name: https://docs.ansible.com/ansible/2.9/modules/os_image_module.html